README.md

```diff
@@ -1,11 +1,11 @@
 RightLink Scripts
 =================
 
-RightScripts for RightScale's RightLink10 (aka RightLinkLite) agent used in the
-Base ServerTemplate and beyond.
+RightScripts for RightScale's RightLink10 agent used in the Base ServerTemplate and beyond.
 
-This repository contains the collection of RightScrits used in ServerTemplates that go with
-the new RightLink10 agent. The scripts for the base ServerTemplate are in the `rightlink` subdirectory.
+This repository contains the collection of RightScripts used in ServerTemplates that go with
+the new RightLink10 agent. The scripts for the base Linux ServerTemplate are in the
+`rll` subdirectory.
```

Comment:
No, it is to distinguish between RightLink Linux (rll) and RightLink Windows (rlw)